common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -48,19 +41,233 @@ Inclusion and Exclusion Fences are defined easily in Mission Planner under the P
\ No newline at end of file
+[wiki site="plane"]
+
+When in Mode FBWB or CRUISE, Plane will attempt to limit altitude target changes by the pilot (ie using Pitch stick for climb/descent rate demand inputs) to avoid breaching a maximum or minimum altitude fence.
+[/site]
+
+.. note:: altitude control imperfections can still result in a breach unless the :ref:`FENCE_MARGIN<FENCE_MARGIN>` param is loose enough to accommodate these imperfections.
+
+
+Enabling Fences in Mission Planner
+==================================
+
+To enable Fences, go to the Mission Planner full parameter list (CONFIG->Full Parameter Tree), search for items with ``FENCE_``:
+
+.. image:: ../../../images/fence_enable.png
+    :target: ../_images/fence_enable.png
+
+
+
+..  youtube:: yhNrtTERnJk
+    :width: 100%
+
+
+[wiki site="copter"]
+
+Copter GeoFence Tab
+-------------------
+
+Copter and Traditional Heli, when connected to Mission Planner, present a CONFIG tab called GeoFence which provides another way to access these parameters in those vehicles.
+
+.. image:: ../../../images/Fence_MPSetup.png
+    :target: ../_images/Fence_MPSetup.png
+
+[/site]
+
+Use for R/C Training
+====================
+
+One of the main uses of geo-fencing is to teach yourself (or someone
+else) to fly radio controlled vehicles. When you have a properly
+configured geo-fence it is very hard to crash, and you can try
+manoeuvres that would normally be too likely to end in a crash, trusting
+the autopilot to 'bounce' the vehicle off the geo-fence before the flight ends
+in disaster.
+
+Geo-fencing can be combined with any flight mode. So for a raw
+beginner, you would combine it with one of the stabilised flight modes
+. Once the pilot has gained some confidence you could combine it with ACRO mode in Copter or MANUAL mode in Plane, which gives direct control of the vehicle and allows for the most interesting aerobatic manoeuvres. 
+
+[wiki site="plane"]
+
+Stick-mixing on fence breach
+============================
+
+ArduPilot enables 'stick mixing' by default when in auto modes. This means
+that you can change the path of a loiter, for example, by using your
+transmitter sticks.
+
+When you are using geo-fencing, stick mixing will be disabled on fence
+breach until your plane is back inside the fenced region. This is to
+ensure that the bad control inputs that caused you to breach the fence
+don't prevent it from recovering to the return point.
+
+As soon as you are back inside the fence stick mixing will be
+re-enabled, allowing you to control the GUIDED mode that the plane will
+be in. If by using stick mixing you manage to take the plane outside the
+fence again then stick mixing will again be disabled until you are back
+inside the fence.
+
+[/site]
+
+MAVLink support
+===============
+
+ArduPilot will report the fence status via the MAVLink GCS protocol. The
+key status packet is called FENCE_STATUS, and is defined in
+"ardupilotmega.xml". A typical FENCE_STATUSpacket looks like this:
```

Comment:
need a space after FENCE_STATUS and before packet.

Reply:
yep